### DISCUSSION TITLE
curl: (22) The requested URL returned error: 400 Bad Request

### DISCUSSION THREAD

**Discussion by aptypic** [deckhouse/deckhouse#4511] [Q&A]
Прошу помощи, во внутреннем контуре в закрытой сети на астра линуксе необходимо развернуть deckhouse
Использую Harbor. Возникает ошибка 400.
Скрипт deckhouse обращается по несуществующему пути https://astrapc.local/v2/deckhouse/manifests/sha256:1f58f948bbc805133cf59a16fd56994df76ee75ad5f2c6824d030e02a59e87f0
Образ находится по пути:
astrapc.local:80/deckhouse/sha256:1f58f948bbc805133cf59a16fd56994df76ee75ad5f2c6824d030e02a59e87f0
[step (1).log](https://github.com/deckhouse/deckhouse/files/11342365/step.1.log)
прилагаю файл лога

**Comment by nabokihms** [comment]
@aptypic hello! Could you please ask your question our support engineers in the telegram chat t.me/deckhouse_ru

On GitHub, we can only offer support in English.
